<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="10" passed="9" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-03-04T00:46:58 ART" name="Default suite" finished-at="2021-03-04T00:47:38 ART" duration-ms="40059">
    <groups>
    </groups>
    <test started-at="2021-03-04T00:46:58 ART" name="Default test" finished-at="2021-03-04T00:47:38 ART" duration-ms="40059">
      <class name="testCases.S001LogInCases">
        <test-method is-config="true" signature="startConnection()[pri:0, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:46:50 ART" name="startConnection" finished-at="2021-03-04T00:46:58 ART" duration-ms="7740" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startConnection -->
        <test-method signature="welcomeisDisplayed()[pri:-2, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:46:58 ART" name="welcomeisDisplayed" description="First screen &#039;Welcome to MYX&#039; is displayed" finished-at="2021-03-04T00:46:58 ART" duration-ms="239" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [WELCOME TO MYXfitness!dsedsf] but found [WELCOME TO MYXfitness!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [WELCOME TO MYXfitness!dsedsf] but found [WELCOME TO MYXfitness!]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at testCases.S001LogInCases.welcomeisDisplayed(S001LogInCases.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- welcomeisDisplayed -->
        <test-method signature="theRoadisDisplayed()[pri:-1, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:46:59 ART" name="theRoadisDisplayed" description="First screen &#039;The road...&#039; is displayed" finished-at="2021-03-04T00:46:59 ART" duration-ms="177" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- theRoadisDisplayed -->
        <test-method signature="invalidEmail()[pri:0, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:46:59 ART" name="invalidEmail" description="Invalid login scenario with a wrong email." finished-at="2021-03-04T00:47:05 ART" duration-ms="5657" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidEmail -->
        <test-method signature="invalidPassword()[pri:1, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:47:05 ART" name="invalidPassword" description="Invalid login scenario with a wrong password." finished-at="2021-03-04T00:47:09 ART" duration-ms="4388" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidPassword -->
        <test-method signature="noEmailAndPassword()[pri:2, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:47:09 ART" name="noEmailAndPassword" description="Invalid login scenario with an empty email and password." finished-at="2021-03-04T00:47:13 ART" duration-ms="3571" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- noEmailAndPassword -->
        <test-method signature="passwordRecoveryEmpty()[pri:3, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:47:13 ART" name="passwordRecoveryEmpty" description="Customer wants to reset the password, but entered no email." finished-at="2021-03-04T00:47:17 ART" duration-ms="3629" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- passwordRecoveryEmpty -->
        <test-method signature="passwordRecoveryWrongEmail()[pri:4, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:47:17 ART" name="passwordRecoveryWrongEmail" description="Customer wants to reset the password, but entered an invalid email." finished-at="2021-03-04T00:47:24 ART" duration-ms="7346" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- passwordRecoveryWrongEmail -->
        <test-method signature="passwordRecoveryzValid()[pri:5, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:47:24 ART" name="passwordRecoveryzValid" description="Customer wants to reset the password, but entered no email." finished-at="2021-03-04T00:47:30 ART" duration-ms="5591" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- passwordRecoveryzValid -->
        <test-method signature="iDontHaveAccount()[pri:6, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:47:30 ART" name="iDontHaveAccount" description="Customer selects the option &#039;I don&#039;t have an account&#039;." finished-at="2021-03-04T00:47:32 ART" duration-ms="2654" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iDontHaveAccount -->
        <test-method signature="validEmailAndPassword()[pri:7, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:47:32 ART" name="validEmailAndPassword" description="Valid login scenario with a valid email and password." finished-at="2021-03-04T00:47:38 ART" duration-ms="5966" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validEmailAndPassword -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:testCases.S001LogInCases@7ed9499e]" started-at="2021-03-04T00:47:38 ART" name="TearDown" finished-at="2021-03-04T00:47:38 ART" duration-ms="79" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- testCases.S001LogInCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
